storage.bs

```diff
@@ -438,10 +438,11 @@ standard and standards using this standard can access the contents.
    <a for="traversable navigable">storage shed</a>.
   </ol>
```

Comment:
You removed _environment_ but here we are trying to get something from _environment_.

Reply:
Good catch!
I have removed the dependency.  Please take another look.